modmain.lua

```diff
@@ -437,7 +445,7 @@ local function KrampedPostInit(self)
 		self.inst:PushEvent("naughtydelta")
 	end
 end
-if SHOWNAUGHTINESS then
+if SHOWNAUGHTINESS and DST == false then -- only activate in DS

```

Comment:
I'd prefer not DST and DST in place of DST == false and DST == true

Testing against exact boolean values like this implies that this variable should not be treated like a boolean but rather stores separate information for boolean and non-boolean values (such as nil).

(please update other places too)

Reply:
Good point. 
Done.